t/e2e/transactions/transactions_test.go

```diff
@@ -40,162 +25,6 @@ type TransactionsTestSuite struct {
-		s.FailNow("sending transaction timed out")
-	}
-
-	s.Equal(`{"jsonrpc":"2.0","id":1,"result":"`+signResult.Response.Hash().Hex()+`"}`, result)
-}
-
-func (s *TransactionsTestSuite) TestCallRPCSendTransactionUpstream() {
-	CheckTestSkipForNetworks(s.T(), params.MainNetworkID, params.StatusChainNetworkID)
-
-	addr, err := GetRemoteURL()
-	s.NoError(err)
-	s.StartTestBackend(e2e.WithUpstream(addr))
-	defer s.StopTestBackend()
-
-	err = s.Backend.SelectAccount(TestConfig.Account2.Address, TestConfig.Account2.Password)
-	s.NoError(err)
-
-	transactionCompleted := make(chan struct{})
-
-	var signResult sign.Result
-	signal.SetDefaultNodeNotificationHandler(func(rawSignal string) {
-		var signalEnvelope signal.Envelope
-		err := json.Unmarshal([]byte(rawSignal), &signalEnvelope)
-		s.NoError(err)
-
-		if signalEnvelope.Type == signal.EventSignRequestAdded {
-			event := signalEnvelope.Event.(map[string]interface{})
-			txID := event["id"].(string)
-
-			// Complete with a wrong passphrase.
-			signResult = s.Backend.ApproveSignRequest(txID, "some-invalid-passphrase")
-			s.EqualError(signResult.Error, keystore.ErrDecrypt.Error(), "should return an error as the passphrase was invalid")
-
-			// Complete with a correct passphrase.
-			signResult = s.Backend.ApproveSignRequest(txID, TestConfig.Account2.Password)
-			s.NoError(signResult.Error, "cannot complete queued transaction %s", txID)
-
-			close(transactionCompleted)
-		}
-	})
-
-	result := s.Backend.CallRPC(`{
-		"jsonrpc": "2.0",
-		"id": 1,
-		"method": "eth_sendTransaction",
-		"params": [{
-			"from": "` + TestConfig.Account2.Address + `",
-			"to": "` + TestConfig.Account1.Address + `",
-			"value": "0x9184e72a"
-		}]
-	}`)
-	s.NotContains(result, "error")
-
-	select {
-	case <-transactionCompleted:
-	case <-time.After(time.Minute):
-		s.FailNow("sending transaction timed out")
-	}
-
-	s.Equal(`{"jsonrpc":"2.0","id":1,"result":"`+signResult.Response.Hash().Hex()+`"}`, result)
-}
-
-func (s *TransactionsTestSuite) TestEmptyToFieldPreserved() {
-	CheckTestSkipForNetworks(s.T(), params.MainNetworkID)
-
-	s.StartTestBackend()
-	defer s.StopTestBackend()
-
-	EnsureNodeSync(s.Backend.StatusNode().EnsureSync)
-	err := s.Backend.SelectAccount(TestConfig.Account1.Address, TestConfig.Account1.Password)
-	s.NoError(err)
-
-	transactionCompleted := make(chan struct{})
-	signal.SetDefaultNodeNotificationHandler(func(rawSignal string) {
-		var sg struct {
-			Type  string
-			Event json.RawMessage
-		}
-		err := json.Unmarshal([]byte(rawSignal), &sg)
-		s.NoError(err)
-		if sg.Type == signal.EventSignRequestAdded {
-			var event signal.PendingRequestEvent
-			s.NoError(json.Unmarshal(sg.Event, &event))
-			args := event.Args.(map[string]interface{})
-			s.NotNil(args["from"])
-			s.Nil(args["to"])
-			signResult := s.Backend.ApproveSignRequest(event.ID, TestConfig.Account1.Password)
-			s.NoError(signResult.Error)
-			close(transactionCompleted)
-		}
-	})
-
-	result := s.Backend.CallRPC(`{
```

Comment:
This test is actually different. We can rewrite it to use `s.Backend.SendTransaction()` with only `from` defined in the params.

Reply:
Addressed